sklearn/feature_selection/_rfe.py

```diff
@@ -714,12 +801,23 @@ def fit(self, X, y, groups=None):
 
             .. versionadded:: 0.20
```

Comment:
Perhaps necessary to also mention that groups can only be passed if `enable_metadata_routing=False`

```
``groups`` can only be passed if metadata routing is not enabled
            via ``sklearn.set_config(enable_metadata_routing=True)``. When routing
            is enabled, pass ``groups`` alongside other metadata via the ``params``
            argument instead. E.g.:
            ``fit(..., params={'groups': groups})``.
```

Reply:
E.g. this seems in line w/ what's in the codebase rn: https://github.com/scikit-learn/scikit-learn/blob/2b2e2903e5635dd93a741c955a87260fb69cfc3d/sklearn/model_selection/_validation.py#L154-L164

Reply:
Yes it does seem that way but for those functions we don't have any kwargs like **params. We have a parameter called params. In this case we are just mentioning groups specifically but it will still be passed in like other params.